packages/dev/sharedUiComponents/src/lines/floatLineComponent.tsx

```diff
@@ -155,26 +155,31 @@ export class FloatLineComponent extends React.Component<IFloatLineComponentProps
     }
 
     incrementValue(amount: number) {
-        if (this.props.step) {

```

Comment:
wait this is a breaking change

Reply:
I mean this is called by the InputArrowsComponent will then ignore the steps if we remove this line

Reply:
Good point, I missed that that was called somewhere else as well.
As a side note, where in the inspector is the `InputArrowsComponent` used? I couldn't find a usage to test if everything worked.